README.md

```diff
@@ -1,32 +1,95 @@
-# RebuildBlackBusiness.com
+<h1 align="center">Welcome to Rebuild Black Business 👋</h1>
+<p>
+  <img alt="Version" src="https://img.shields.io/badge/version-1.0.0-blue.svg?cacheSeconds=2592000" />
+  <a href="https://twitter.com/ThugDebugger" target="_blank">
+    <img alt="Twitter: ThugDebugger" src="https://img.shields.io/twitter/follow/ThugDebugger.svg?style=social" />
+  </a>
+</p>
 
-## Local Development
+> The website for https://www.rebuildblackbusiness.com/
 
-### 1. Clone the repository
+### 🏠 [Homepage](https://www.rebuildblackbusiness.com/)
 
-First, fork this repository by clicking the “fork” button at the top right. This creates your own copy of the repository where you’re able to make changes.
+## Mission
 
-Second, clone your copy of the repo to your local machine:
+## 🤝 Contributing
 
-```bash
-# clone the repo
-git clone git@github.com:YOUR_USERNAME_HERE/RBB-Website.git
+Contributions, issues and feature requests are welcome!<br />Feel free to check [issues page](https://github.com/Rebuild-Black-Business/RBB-Website/issues).
+
+### How can I get involved now?
+
+We're tracking all high level Web Development tasks in Trello:
+
+- [Front-End Development Trello](https://trello.com/b/d1sguaj6/rebuild-black-business-frontend-dev)
+- [Back-End Development Trello](https://trello.com/b/034hEV4X/rebuild-black-business-backend-dev)
```

Comment:
Feel free to ignore this, but don't GitHub Issues/Projects do everything Trello’s doing here in a way that doesn't require signing up for additional accounts? Another benefit of using issues is that they can be referenced from pull requests to add context, auto-move between swim lanes, etc.

Reply:
💯 - I think we'll see what we can do about using GitHub as a more single source of truth for the roadmap and current workload. Trello is an artifact of just trying to get up and running quickly. We'll simplify this as quickly as we can!

Reply:
Now that we have a repo I'd be happy to start creating new pieces of work in GitHub